trollmoves/movers.py

```diff
@@ -201,13 +211,38 @@ class FtpMover(Mover):
     active_connections = dict()
     active_connection_lock = Lock()
 
+    def _get_netrc_authentication(self):
+        """Get login authentications from netrc file if available"""
+        LOGGER.debug("HOME = %s", os.environ.get('HOME'))
+        try:
+            secrets = netrc.netrc()
+        except (netrc.NetrcParseError, FileNotFoundError) as e__:
+            LOGGER.warning('Failed retrieve authentification details from netrc file! Exception: ', str(e__))
+            return
+
+        LOGGER.debug("hosts: %s", str(list(secrets.hosts.keys())))
+        LOGGER.debug("Check if hostname matches any listed in the netrc file")
+        if self.destination.hostname in list(secrets.hosts.keys()):
+            LOGGER.debug("Destination hostname: %s", self.destination.hostname)
+            self._dest_username, account, self._dest_password = secrets.authenticators(self.destination.hostname)
+            LOGGER.debug("Destination hostname: %s", self.destination.hostname)
```

Comment:
This log message is done twice.

Reply:
Oh, yes, it is probably quite verbose now, as I needed some indications on where it crached until I found the error... I will reduce!